data/po/pl.po

```diff
@@ -243,10 +243,9 @@ msgstr "Pokaż FPS w grach:"
 
 #. Has to end with ": "
 #: data/ui/preferences.ui:201
-#, fuzzy
 msgctxt "show_windows_games"
 msgid "Show Windows games: "
-msgstr "Pokaż FPS w grach:"
+msgstr "Pokaż FPS w grach: "
```

Comment:
I think this should be changed. It still contains FPS

Reply:
You're right . I don't know how it could get there :)